props/nested-components.md

```diff
@@ -0,0 +1,195 @@
+
+## Presentational Components
+
+Presentational components are components that are responsible for displaying data.
+
+Presentational components are not responsible for modifying data.
+
+## Container Components
+
+Container components are components responsible for:
+
+- Using presentation components
+- Holding data
+- Modifying and managing data
+- Sending data to presentation components
+
+Container components often have presentation components _nested_ inside them.
+
+## Example: `StudentList` and `Student` Components
+
+Sofia is a teacher creating an attendance app. Her app should display a list of students. At this moment, she starts with this implementation of a `StudentList` component in `src/components/StudentList.js`.
+
+(Class names have been omitted from this code snippet for clarity)
+
+<!-- prettier-ignore-start -->
+```js
+import React from 'react';
+import './StudentList.css';
+
+const StudentList = () => {
+    return (
+        <section>
+            <h2>Student List</h2>
+            <ul>
+                <li>Student A</li>
+                <li>Student B</li>
+                <li>Student C</li>
+            </ul>
+        </section>
+    )
+}
+
+export default StudentList;
+```
+<!-- prettier-ignore-end -->
+
+This component displays a list of students, where each student is in an `<li>` element.
+
+However, Sofia knows that each student list item should display a lot of data, such as their name and email address.
+
+Sofia can apply the container and presentational component design patterns, and:
+
+1. Create a `Student` component, which will act as a presentational component
+1. Use the `Student` component in `StudentList`, which will act as a container component
+
+### `Student` Component
+
+Sofia's `Student` component should display a student's name and email address. She'll create a new file, `src/components/Student.js`, and write this code:
+
+<!-- prettier-ignore-start -->
+```js
+import React from 'react';
+
+const Student = () => {
+    return (
+        <li>
+            <ul>
+                <li>Nickname: Ada</li>
+                <li>Email: ada@dev.org</li>
+            </ul>
+        </li>
+    )
+}
+
+export default Student;
+```
+<!-- prettier-ignore-end -->
+
+### Rendering `Student`s in `StudentList` Component
+
+Now, Sofia can update her `StudentList` component.
+
+First, she needs to import the `Student` component into `src/components/StudentList.js` with `import Student from './Student';`.
+
+Then, she should replace the contents of the `<ul>` element.
+
+<!-- prettier-ignore-start -->
+```js
+const StudentList = () => {
+    return (
+        <section>
+            <h2>Student List</h2>
+            <ul>
+                <Student></Student>
+                <Student></Student>
+                <Student></Student>
```

Comment:
leave the `<li>` elements here so that Student doesn't need to be aware of where it's being rendered
```suggestion
                <li><Student></Student></li>
                <li><Student></Student></li>
                <li><Student></Student></li>
```